packages/components/panel/src/panel.scss

```diff
@@ -14,11 +14,11 @@
 }
 
 :host([elevation='raised']) {
-  background: var(--sl-elevation-surface-raised-alternative-idle);
```

Comment:
Can you please also update this token in the `All` story?

<img width="687" alt="image" src="https://github.com/user-attachments/assets/094176e3-9f31-475f-b4ab-a4d47c346aee" />